packages/manager/modules/vps/src/dashboard/migrate/migrate.routing.js

```diff
@@ -31,15 +32,22 @@ export default /* @ngInject */ ($stateProvider) => {
         $state,
         atInternet,
         migrationTrackingPrefix,
-      ) => () => {
+      ) => (selectedPlan) => {
         atInternet.trackClick({
           name: `${migrationTrackingPrefix}::confirm`,
           type: 'action',
         });
-        return $state.go('vps.detail.dashboard.migrate.confirm');
+        return $state.go('vps.detail.dashboard.migrate.confirm', {
+          selectedPlan,
+        });
+      },
+      newPlans: /* @ngInject */ (catalog, vpsMigration) => {
+        const catalogP = [];
+        angular.forEach(vpsMigration.plans, (plans) =>
+          catalogP.push(find(catalog.plans, { planCode: plans.newPlan })),
+        );
```

Comment:
Prefer using native JS function as possible is duable.
```suggestion
        vpsMigration.plans.forEach((plans) =>
          catalogP.push(catalog.plans.find(({planCode}) =>  plans.newPlan === planCode),
        );
```

Reply:
Added the suggested changes to the PR.